webpack/loaders.js

```diff
@@ -23,14 +13,18 @@ export default (appName) => {
                 'transform-runtime',
                 'add-module-exports',
             ],
-            presets,
+            presets: [
+                'es2015',
+                'react',
+                'stage-0',
+            ],
         },
     }, {
         test: /\.json$/,
         loader: 'json-loader',
     }, {
         test: /\.jpe?g$|\.gif$|\.png$/,
-        loader: `url-loader?limit=10000&name=${appName}/[hash].[ext]`,
+        loader: `url-loader?limit=10000&name=/${appName}/[hash].[ext]`,
```

Comment:
We need to have `/[hash].[ext]` for the frontend and `/admin/[hash].[ext]` for admin.